src/app/client/src/app/modules/shared/components/redirect/redirect.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="ui center aligned">
+    <p class="segmentCardSubDesc"> As the Content is from an External URL the content will be opened in this Tab </p>
```

Comment:
Texts should not be hardcoded.

Reply:
Since yesterday, resourceServices is not working, so we were getting error, if resourceService is working fine, the hardcoded message will be removed

Reply:
Added in resourcebundle